### DISCUSSION TITLE
Daily Standup (Tuesday, June 8th)

### DISCUSSION THREAD

**Discussion by DavBot02** [CarlosCRG19/ijalti-peers#107] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished first version #99 
 - Started working on #108

**What are your priorities for the next 24 hours?**:
 - Keep working on it

**Blockers**:
 - Some MUI issues

**Shoutouts**:
 - @BrandonMagana i like him

**Reply by BrandonMagana** [reply]
**What did you achieve in the last 24 hours?**:
 - Complete aspirant filtering feature

**What are your priorities for the next 24 hours?**:
 -  hacerme baqueton

**Blockers**:
 - None

**Shoutouts**:
- @CristianCazares for search aspirant view

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Almost finished the flippin aspirant search

**What are your priorities for the next 24 hours?**:
 - Finish it, merge it and take on new task

**Blockers**:
 - A little bug that does not allow me to show the selected option of education

**Shoutouts**:
 - @DavBot02 @antonieto @BrandonMagana

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Coded an endpoint for company serach.

**What are your priorities for the next 24 hours?**:
 - Implement searchbar in frontend 
 - Start working on interested in feature

**Blockers**:
 - None

**Shoutouts**:
 - @CristianCazares and @DavBot02 cause they are handsome
